static/src/js/util/map/getQueryFromShapefileFeature.ts

```diff
@@ -2,10 +2,27 @@ import { Feature } from 'ol'
 import { Point } from 'ol/geom'
 
 import spatialTypes from '../../constants/spatialTypes'
-import { SpatialQueryType } from '../../types/sharedTypes'
+import { Spatial, SpatialQueryType } from '../../types/sharedTypes'
+
+// Coordinate types for spatial types
+type Coordinate = [number, number] | [number, number, number]
```

Comment:
Maybe move these to sharedTypes.ts? They should have ts style (`/** **/`) comments as well